plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherExtension.kt

```diff
@@ -3,13 +3,17 @@ package com.github.triplet.gradle.play
 import com.github.triplet.gradle.play.internal.AccountConfig
 import com.github.triplet.gradle.play.internal.ReleaseStatus
 import com.github.triplet.gradle.play.internal.TrackType
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
 
 open class PlayPublisherExtension : AccountConfig by PlayAccountConfigExtension() {
+    @get:Internal
```

Comment:
Explain that this is a backing property via annotation value? (see above)

Reply:
Done